massdriver/resource_package_alarm.go

```diff
@@ -0,0 +1,90 @@
+package massdriver
+
+import (
+	"context"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type PackageAlarmMetadata struct {
+	ResourceIdentifier string `json:"resource_identifier"`
+	DisplayName        string `json:"display_name"`
+}
+
+func resourcePackageAlarm() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourcePackageAlarmCreate,
+		ReadContext:   schema.NoopContext,
+		UpdateContext: schema.NoopContext,
+		DeleteContext: resourcePackageAlarmDelete,
+
+		Schema: map[string]*schema.Schema{
+			"resource_identifier": {
```

Comment:
Related to above, should we call this something to the effect of `cloud_resource_id` ... I was confused as to what it was until I got to this description.